src/components/Button/Button.tsx

```diff
@@ -87,7 +86,7 @@ const baseStyles = ({ theme }: StyleProps) => css`
 
 const primaryStyles = ({
   theme,
-  variant = 'primary'
+  variant = 'secondary'
```

Comment:
While I understand the purpose, this isn't very intuitive, is it?

Reply:
Do you mean this particular piece of code or the fact that I changed the default `variant` to `secondary`?

Reply:
Having the default variant as secondary

Reply:
Yeah, I agree. That's why I changed it to `primary` initially. Then I realized that it would be a **very** painful migration to change this. Previously, the default button...

```tsx
<Button />
```

...would render a secondary button. It's easiest to keep it this way.

Reply:
What would be your suggestion here?

Reply:
Honestly none. I agree it would cause more trouble currently. Let’s ship it this way